packages/typespec-vscode/src/types.ts

```diff
@@ -9,6 +10,7 @@ export const enum CommandName {
   InstallGlobalCompilerCli = "typespec.installGlobalCompilerCli",
   CreateProject = "typespec.createProject",
   OpenUrl = "typespec.openUrl",
+  Generate = "typespec.generate",
```

Comment:
typespec.emitCode?   it's better to name it in format verb+name. It's hard for people to figure out what it means by just a verb 'generate'
also i saw you are using "emit" all the places in code, so it's better to have the command name align with it.

Reply:
As we discussed, we will not only generate code, but maybe others, so in the command title we use `TypeSpec: Generate from TypeSpec`, ignore `code`, so I think the command name should consistent.

Reply:
update to `GenerateCode = "typespec.generateCode"`